R/messages.R

```diff
@@ -1,21 +1,14 @@
 messages <- list(
   errorWrongType = function(objectName, type, expectedType, optionalMessage = NULL) {
-    # Name of the calling function
-    callingFunctions <- sys.calls()
-    callingFunction <- sys.call(-length(callingFunctions) + 1)[[1]]
-
+    callingFunction <- .getCallingFunctionName()

```

Comment:
I moved the code for checking for the name of the calling function to a separate function to reduce code duplication.